app/js/components/Draft.jsx

```diff
@@ -2,17 +2,50 @@ import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import shortid from 'shortid';
+import constants from '../utils/constants';
+import IconButton from './button/IconButton';
 
 export class Draft extends PureComponent {
   renderDraftList = () => {
     const { draftOrders } = this.props;
     return draftOrders.map((order) => {
+      const isPanel = !!order.set;
+      const draftType = !isPanel ? 'single' : 'panel';
       const orderName = order.display || order.drugName;
+      const iconClass = classNames(
+        'scale',
+        {
+          'i-gray': order.urgency === constants.ROUTINE || typeof order.urgency === 'undefined',
+          'i-red': order.urgency === constants.STAT,
+        },
+      );
+
       return (
         <li className="draft-list small-font" key={shortid.generate()}>
+          <span className="order-status">{!order.action ? 'NEW' : order.action}</span>
           <span className="draft-name">{orderName.toLowerCase()}</span>
           <div className="action-btn-wrapper">
-            {/* The action are to be dynamically set */}
+            <span className="action-btn">
+              { order.type !== 'drugorder' ?
+                <a
+                  id="draft-toggle-btn" href="#" onClick={() => {}}
+                >
+                  <i className={iconClass} title="Urgency">&#x25B2;</i>
+                </a> :
+                <IconButton
+                  iconClass="icon-pencil"
+                  iconTitle="EDIT"
+                  onClick={() => {}}
+                />
+              }
+            </span>
+            <span className="action-btn right">
+              <a
+                id="draft-discard-btn" href="#" onClick={() => {}}
+              >
+                <i className="icon-remove scale" title="Discard" />
+              </a>
```

Comment:
I see you also composed the Discard icon with 
```
<a
  id="draft-discard-btn" href="#" onClick={() => {}}
>
 <i className="icon-remove scale" title="Discard" />
</a>
```
Is there a reason why you are not using the `IconButton` component created for this purpose?

Reply:
Thanks, Corrected.